src/loaders/utils/relationsAndColumnsSystemCatalogQuery.ts

```diff
@@ -217,8 +217,11 @@ export type RawRelationsAndColumnsRow = RawRelationRow | RawColumnRow | RawViewD
  * 1. Sort rows by relation (table) name ASC, then by column number ASC with null (relation rows) first.
  * 2. Iterate in order, creating a new Relation when a relation row is encountered, then attaching its columns.
  * 3. Column rows without a preceding relation row are cause for an error.
+ *
+ * The provided `database` is used to add parent resource context to each relation.
```

Comment:
This is a pretty common pattern we have for other resource types, and helps `FlinkRelation` play nicely with the [`FlinkDatabaseResource` union type](https://github.com/confluentinc/vscode/blob/27ef8b63db796f9efea1168ed4496cd85b3aab13/src/models/flinkDatabaseResource.ts#L12).